rules/manager.go

```diff
@@ -871,7 +903,7 @@ func (m *Manager) RuleGroups() []*Group {
 
 	// Sort rule groups by file, then by name.
 	sort.Slice(rgs, func(i, j int) bool {
-		if rgs[i].file != rgs[j].file {
+		if rgs[i].name == rgs[j].name {
```

Comment:
It kinda seems odd to sort by file as if it's lower in the file->group->rule hierarchy than the group... how about always sorting by file first, then by group name, but surfacing the file name in the group header somehow (maybe `{{.Name}} ({{.File}})` instead of just `{{.Name}}`)? I don't think we should introduce a new dropdown level in the UI for just the file, but making it visible in an unobtrusive way could be good and make the file->group sorting behavior less confusing for users. Maybe even make the group name grey/italic/small or something, so it pops out less, but is still there?

Reply:
Hmm, your screenshots and updates say that the ordering now happens by file first, but the code here still looks like it's sorting by name first?

Reply:
Ah, I think a code push is missing?